Gemfile

```diff
@@ -29,6 +29,7 @@ gem 'rubocop-rails', '~> 2.8.1'
 
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.2', require: false
+gem 'faker', git: 'https://github.com/faker-ruby/faker.git', branch: 'master'
```

Comment:
move it to :development, :test group, you don't need it on production I believe

Reply:
or if you will run seeds with faker on production, you can leave like it is

Reply:
Resolve